templates/store.html

```diff
@@ -39,7 +39,7 @@ <h4 class="p-muted-heading u-float-left">{{ results|length }} items</h4>
           </select>
         </div>
       </div>
-      <div class="row">
+      <div class="row u-equal-height">
```

Comment:
this doesn't seem to do anything?

Reply:
I'm just trying to get my characters-per-file count up! Will remove, thanks!